packages/api/package.json

```diff
@@ -10,20 +10,20 @@
   "types": "./dist/index.d.ts",
   "license": "MIT",
   "dependencies": {
-    "@graphql-tools/merge": "7.0.0",
+    "@graphql-tools/merge": "8.1.2",
```

Comment:
Not sure we need any graphql related dependencies anymore in api. I think I will remove and test,

Reply:
```suggestion
```